packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -372,102 +360,184 @@ impl PostgresDatabase {
     }
```

Comment:
(can't comment at the right location, but line 347, function `get_by_version`) If we're drive-by changing to the other query syntax, the function here is still using `fetch_one` with an executor. Should we change it here as well?

Reply:
Good catch: 2b642138efd2ea5e92b21f21a7e836a07822d01e